### PR TITLE
Review and update Validations doc

### PR DESCRIPTION
**Pull request by lread**
Re-organize and re-section.

Add some info on exit codes.

Add a screenshot to show an example of `ws get:messages`.

Update Warning 206 screenshot to what it should be now: Error 111.

Turf note on Warning 207 that it is only be applied to check command, it's not true anymore.

Configuration doc:
- the only reason we documented `:bricks` was to describe disabling Error 111, this is no longer a feature so we no longer need to describe `:bricks`.
- correct  description for `:necessary` and add link to Validations doc.

Edits for clarity.

### CODE REVIEW

**Comment by lread** [comment]
@tengstrand, ready for review and feedback.
As always, please read the commit message first.

File: doc/configuration.adoc [hunk 1]
```diff
@@ -89,10 +89,6 @@ The workspace map keys are:
 |===
 | Key | Description
 
-a| `:bricks`
```
Comment (tengstrand) [inline doc/configuration.adoc:92]: We need to keep this one, and add information about the `:keep-lib-versions` key, which is described in libraries.adoc. Probably a link to the library section is enough.

File: doc/configuration.adoc [hunk 2]
```diff
@@ -168,7 +164,8 @@ You can use the alias instead of the full project name for various `poly` comman
 If not specified, the xref:commands.adoc#info[info] and xref:commands.adoc[libs] commands emit a `?` followed by a number (e.g. `?1`) for project column headings.
```
Comment (tengstrand) [inline doc/configuration.adoc:172]: We need to add `:keep-lib-versions` here (that I missed). We could add the key to the example above also.

File: doc/configuration.adoc [hunk 3]
```diff
@@ -168,7 +164,8 @@ You can use the alias instead of the full project name for various `poly` comman
...
 
 a| [nowrap]`:necessary`
-a| Supports turning off `Error 111` for one or more bricks.
+a| Supports turning off `Warning 207` for one or more bricks.
```
Comment (tengstrand) [inline doc/configuration.adoc:177]: Good catch!

**Review by tengstrand — CHANGES_REQUESTED** [review]

File: doc/configuration.adoc (hunk 1 above)
Comment (lread) [reply doc/configuration.adoc:92]: Aha, ok, will do.

File: doc/configuration.adoc (hunk 2 above)
Comment (lread) [reply doc/configuration.adoc:172]: Righty-o, will do.

File: doc/configuration.adoc (hunk 3 above)
Comment (lread) [reply doc/configuration.adoc:177]: tx

File: doc/configuration.adoc (hunk 1 above)
Comment (lread) [reply doc/configuration.adoc:92]: Done!

File: doc/configuration.adoc (hunk 2 above)
Comment (lread) [reply doc/configuration.adoc:172]: Done!

**Comment by lread** [comment]
@tengstrand review feedback addressed.

**Review by tengstrand — APPROVED** [review]

**Pull request merged**